masterlist.yaml

```diff
@@ -8741,6 +8741,53 @@ plugins:
     after: [ 'SkyTEST-RealisticAnimals&Predators.esp' ]
     tag: [ Text ]
 
+  - name: 'Apothecary.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/52130/' ]
+    after: [ 'TechnicolorAlchemy.esp' ]
```

Comment:
The order doesn't seem to matter for v1.41 of Technicolor Alchemy, & LOOT's algorithm should sort v1.55b this way by default:

```suggestion
```

Reply:
I wish you would please consider leaving this one in.  It actually is the entire reason that I entered all of this.  I figured it would be nice to add the rest too.  I've built Flora Respawn Fix into TA but CACO uses a modified version of the WM Flora Fix which is a more advanced SKSE version.  I just cringe to think what might happen if somehow something were to get wonky if a player added one or the other mid game play.  If you absolutely refuse then that's fine, I was just caught up in an overabundance of caution.

Reply:
That's fine, it can be left then.